src/app/templates/index.ftl

```diff
@@ -25,8 +25,7 @@
 
             </th>
             <#if isReviewer>
-                <th><label for="status-filter">Status</label>
-
+                <th style="width:100px"><label for="status-filter">Status</label>
```

Comment:
Could use em instead of fixed pixels - though it doesn't look like font size is one of the user variables that can be changed, and it all behaves nicely when zoomed, so this shouldn't cause any problems.